.travis.yml

```diff
@@ -63,6 +68,10 @@ install:
 script:
   - ./scripts/ci/test.sh -o $HOST_OS -r $TARGET_ARCH
   - ./scripts/ci/build-installers.sh -o $HOST_OS -r $TARGET_ARCH
+  - if [[ ${TRAVIS_BRANCH} == "master" ]] && [[ ${TRAVIS_PULL_REQUEST} == "false" ]]; then
```

Comment:
Hmmm, so we only update the cache on merge-commits to `master`? Should this also check that `[[ "$TRAVIS_OS_NAME" == "linux" ]]` ?